src/openrct2-android/app/build.gradle

```
@@ -17,7 +17,7 @@ android {
                 arguments '-DANDROID_STL=c++_shared'
                 targets 'openrct2', 'openrct2-ui', 'openrct2-cli'
                 // abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
-                abiFilters 'armeabi-v7a', 'arm64-v8a'
+                abiFilters 'arm64-v8a'
```

```suggestion
                abiFilters 'arm64-v8a', 'x86_64'
```

Should we build both ABIs?